GeeksCoreLibrary/Components/Account/Account.cs

```diff
@@ -1897,12 +1897,14 @@ private async Task SaveGoogleClientIdAsync(ulong userIdForGoogleCid)
             var googleClientId = String.Join(".", clientIdSplit.Skip(2));
 
             var tablePrefix = await wiserItemsService.GetTablePrefixForEntityAsync(Settings.EntityType);
+            
+            var detail = new WiserItemDetailModel()
+            {
+                Key = String.IsNullOrWhiteSpace(Settings.GoogleClientIdFieldName) ? "google-cid" : Settings.GoogleClientIdFieldName,
```

Comment:
In de originele code werd "Constants.DefaultGoogleCidFieldName" gebruikt voor "google-cid", kan je dit weer goed zetten om de constant te gebruiken?

Reply:
whoops yeah lets return that, thanks